### PR TITLE
MYSQL-80 Remove plugin .json file for version 2.0.26 from the code repository, MYSQL-78 Support latest dvp version 3.1.0, MYSQL-59 Security -> Password field on UI can be used to inject SQL queries

### PR DESCRIPTION
**Pull request by gmanhas23**
…ueries

### Problem

1. Remove plugin .json file for version 2.0.26 from the code repository

2. Plugin should be supported with latest dvp version 3.1.0, corresponding to Python 2.7

3. User is able to inject SQL queries using password field in both Linking and Provisioning wizard. Bug created https://delphix.atlassian.net/browse/MYSQL-59

### Solution

1. Remove plugin .json file for version 2.0.26 from the code repository
2. Added a new required schema for DVP 3.1.0 [Snapshot Parameters Definition](https://developer.delphix.com/References/Schemas_and_Autogenerated_Classes/#snapshotparametersdefinition-schema).
3. Disallowed spaces, single quotes and double quotes from the UI for password strings. 

### Testing Done
Tested with the query strings on dsource and VDB creation pages and user is not able to write SQL queries onto the textbox in UI

### CODE REVIEW

**Review by vinaybyrappa — APPROVED** [review]

File: plugin_config.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ pluginType: STAGED
 entryPoint: plugin_runner:plugin
 srcDir: src
 schemaFile: schema.json
-buildNumber: 2.0.26
+buildNumber: 2.0.40
```
Comment (batrankit) [inline plugin_config.yml:10]: I think we agreed on using 2.1.0,right?

File: schema.json [hunk 2]
```diff
@@ -199,8 +200,9 @@
           "sourcePass": {
              "type": "string",
              "format": "password",
+             "pattern" : "^[^\"\\'\\s]*$",
              "prettyName": "Source Connection Password",
-             "description": "* Required if Delphix is taking backups",
+             "description": "* Required if Delphix is taking backups. Double quote, single quote or space characters are not valid.",
```
Comment (batrankit) [inline schema.json:205]: "*" here in the starting of the description. Is that required?

File: schema.json [hunk 3]
```diff
@@ -79,12 +79,13 @@
...
-         },         
+         },
          "vdbPass": {
             "type": "string",
             "format": "password",
+            "pattern" : "^[^\"\\'\\s]*$",
             "prettyName": "DB Password",
-            "description": "DB user password for the virtual database",
+            "description": "DB user password for the virtual database. Double quote, single quote or space characters are not valid.",
             "default": "Delphix@123"
```
Comment (batrankit) [inline schema.json:89]: Do we need to remove this default password, here and at other places.

File: plugin_config.yml (hunk 1 above)
Comment (gmanhas23) [reply plugin_config.yml:10]: I thought we can make that so in the main branch when we do the final merge. Thoughts?

File: schema.json (hunk 2 above)
Comment (gmanhas23) [reply schema.json:205]: It was already there. I guess just as a visual aid to show that this is a required thing. I think it's pretty functional with the current UI wizard where we have so many fields. I think we should keep it

**Review by batrankit — APPROVED** [review]

**Pull request merged**